### ISSUE TITLE
Existing PVC requires editing the template

### ISSUE THREAD

**Issue by kevprice83** [3scale/3scale-amp-openshift-templates#6]
This is not so much an issue but an enhancement request. If the PVC's already exist (will be a common scenario) then the reference to those PVC names needs to be edited in the template because it is very unlikely the same names are used. It would be nice if these could be parameterised. Any thoughts on this one?

**Comment by andrewdavidmackenzie** [comment]
I think that when you run it, if they already exist, then you will get an error message (already created?) but in fact all will work OK and you don't need to edit it?

**Comment by kevprice83** [comment]
@andrewdavidmackenzie yes that is correct up to the point that it will all work, that will only happen if the PVC names are the same as what each of the dc's are referencing in the template. In the case that pre-existing PVCs have different names then those references need to be changed. No biggy but if it is possible to parameterise these values it can make for a smoother deployment experience and less error prone.

**Comment by kevprice83** [comment]
@andrewdavidmackenzie should I move this issue to the open shift repo we have?

**Comment by andrewdavidmackenzie** [comment]
Kevin, this repo is really for "capturing" the output of the productization process that takes the upstream templates and modifies them (e.g. point to productized docker images, etc) and puts them here.

We originally did to have a place to "host" the templates that we could point to with a simple link to download them....

We don't work on the templates here per se...

So, I think we should limit all issues to the upstream repo, they could be issues with those "dev" templates, or with the process that takes the "dev" templates and modifies them to produce these ones here....

Makes sense?
